force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -248,15 +308,19 @@ public with sharing class ClassesObjectsMethods {
      * Question 12
      * Create a Book object and returns the details of the book.
      * For example, createAndGetBookDetails() should return a string like "Title: A Brief History of Time, Author: Stephen Hawking".
+     * 
+     * 
      * @return A string containing the details of the book.
      */
     public static String createAndGetBookDetails() {
-        // Create a Book object with title "A Brief History of Time" and author "Stephen Hawking"
+        // Create a Book object with title "A Brief History of Time" and author "Stephen Hawking"                
+        Book herBook = new Book('A Brief History of Time', 'Stephen Hawking');
 
         // Call the getBookDetails method on the book object to get the details of the book
+        herBook.getBookDetails();
 
         // Return the details of the book
-        return null; // Replace null with the variable you used to store the result
+        return herBook.getBookDetails(); // Replace null with the variable you used to store the result
```

Comment:
Try assigning herBook.getBookDetails() to a new variable and returning that variable